CONTRIBUTING

```diff
@@ -0,0 +1,10 @@
+
+Contributing to the WiredTiger project
+======================================
+
+Pull requests are always welcome, and the WiredTiger dev team appreciates any help the community can
+give to help make WiredTiger better.
+
+For more information about how to contribute, please read `the WiredTiger Wiki on GitHub`_.
+
+.. _the WiredTiger Wiki on GitHub: https://github.com/wiredtiger/wiredtiger/wiki/Contributing-to-WiredTiger
```

Comment:
Is this `.. _` thing the syntax for a footnote? I tried it with a couple of online renderers and it just rendered as-is.

Reply:
Ah - I'm not sure, I copied that from the MongoDB contributor guide verbatim, so I figured it was some odd syntax. Let me see if I can get more confident that it's correct.